topics/c/index.md

```diff
@@ -11,4 +11,4 @@ topic: c
 url: http://www.open-std.org/jtc1/sc22/wg14/
 wikipedia_url: https://en.wikipedia.org/wiki/C_(programming_language)
 ---
-C is a programming language designed by Dennis Ritchie at Bell Labs. C is very widely used, straightforward, and can be compiled to a number of platforms and operating systems. C is an imperative language, with a small number of keywords and a large number of mathematical operators.
+C is a programming language designed by Dennis Ritchie at Bell Labs. C is very widely used, straightforward, and can be compiled to a number of platforms and operating systems. C is an imperative language, with a small number of keywords and a large number of mathematical operators. C is also a very low lavel programming language. That means it can communicate directly with hardware.
```

Comment:
Seems like you might be able to combine these two sentences into one to make it clear that the reasons that C can communicate directly with hardware. What do you think?
```suggestion
C is a programming language designed by Dennis Ritchie at Bell Labs. C is very widely used, straightforward, and can be compiled to a number of platforms and operating systems. C is an imperative language, with a small number of keywords and a large number of mathematical operators. C is also a very low level programming language, which means it can communicate directly with hardware.
```

Reply:
> Seems like you might be able to combine these two sentences into one to make it clear that the reasons that C can communicate directly with hardware. What do you think?

Oh sorry! Made a spell mistake...